src/Report/Html/Renderer/Template/coverage_bar.html.dist

```diff
@@ -1,5 +1,7 @@
-       <div class="progress">
+
```

Comment:
Please avoid unnecessary (whitespace) changes.

Reply:
Are tags supposed to line up? Because this one didn't.

Reply:
I do not know what you mean by that, sorry.

Reply:
The generated source code was messy. It no longer is. Is indentation import to you? Either indent correctly, or have totally random indentation, or no indentation at all. Don't indent everything perfectly except for a couple of tags.

This is how it's currently formatted:
```
      <tr>
       <td class="danger">Total</td>
       <td class="danger big">       <div class="progress">
         <div class="progress-bar bg-danger" role="progressbar" aria-valuenow="0.00" aria-valuemin="0" aria-valuemax="100" style="width: 0.00%">
           <span class="sr-only">0.00% covered (danger)</span>
         </div>
       </div>
</td>
       <td class="danger small"><div align="right">0.00%</div></td>
       <td class="danger small"><div align="right">0&nbsp;/&nbsp;1</div></td>
       <td class="danger big">       <div class="progress">
         <div class="progress-bar bg-danger" role="progressbar" aria-valuenow="46.15" aria-valuemin="0" aria-valuemax="100" style="width: 46.15%">
           <span class="sr-only">46.15% covered (danger)</span>
         </div>
       </div>
</td>
```